src/pykka/_proxy.py

```diff
@@ -199,16 +199,15 @@ def __hash__(self):
         return hash((self._actor, self._attr_path))
 
     def __repr__(self):
-        return f"<ActorProxy for {self.actor_ref}, attr_path={self._attr_path!r}>"
+        return (
+            f"<ActorProxyMessageBuilder for {self.actor_ref}, "
+            f"attr_path={self._attr_path!r}>"
+        )
 
-    def __dir__(self):
```

Comment:
Just out curiosity, why are we changing these two methods? I.e. if you explain the new builders and factory stuff on a high level about how things fit together I think you'll answer this at the same time.